examples/image-carousel/elements/Image.js

```diff
@@ -0,0 +1,52 @@
+const Tram = require('../../../tram-one')
+
+const html = Tram.html()
+
+const divStyle = `
+  width : 100%;
+`
+
+const imgStyle = `
+  width : 100%;
+  height : 400px;
```

Comment:
The fixed height actually is stretching the images, you can just remove this line.